### PR TITLE
Create Installation de bibliothèques Python avec pip

### LINKED ISSUE #1948: Error in command in Python pip tutorial
[Reviewing the FR translation](https://github.com/programminghistorian/ph-submissions/issues/320) of the lesson [Installing Python Modules with pip](https://programminghistorian.org/en/lessons/installing-python-modules-pip) established that the command provided in the[ Windows instructions part](https://programminghistorian.org/en/lessons/installing-python-modules-pip#windows-instructions) of the EN original has an error. 
In the original EN lesson, the command reads currently as follows: ```python get-pip.py to install pip```  but the ```to install pip``` part must have slipped in the code by mistake as it does not correspond to a known command but it rather apparently continues the text (phrase) provided before the code. 
We'll soon be done with the translation review and can fix the error in the EN and ES versions of the lesson once I make the pull request - ~~@ssimpkin~~ @svmelton and @rivaquiroga I will ask for your reviews and perhaps you'll need to intervene to reajust the phrase. Thanks a lot to @fdlaramee for his vigilance. 
I will close this issue with the pull request that will publish the lesson in some days from now.

### PR DESCRIPTION
**Pull request by spapastamkou**
*New FR translated lesson. Closes #1948*

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
  - [x] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [x] Add the appropriate "Label"
- [x] [Ensure the Travis CI checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [x] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing Travis errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Travis Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-travis-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

**Comment by spapastamkou** [comment]
@rivaquiroga @svmelton As per #1948 I fixed the script error in the EN and EN lessons. I tried to adapt the text that preceded the script, so please check if things are ok (I asked for your reviews).

**Comment by spapastamkou** [comment]
@programminghistorian/technical-team There seems to be a weird build fail because of... Twitter links? Can someone please [check](https://travis-ci.com/github/programminghistorian/jekyll/builds/212645482) and help me with this? Thank you.

**Comment by rivaquiroga** [comment]
@spapastamkou, the same happened to me yesterday. You just need to trigger a new build and the problem will eventually go.

File: fr/lecons/installation-modules-python-pip.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+--------------------------
+
+Maintenant que vous disposez de `pip`, il est facile d'installer des bibliothèques Python car `pip` fait tout le travail pour vous. Lorsque vous trouvez une bibliothèque que vous souhaitez utiliser, généralement, la documentation ou les instructions d'installation incluent la commande `pip` nécessaire, telle que :
+
+```bash
+$ pip install requests
+$ pip install beautifulsoup4
+$ pip install simplekml
+```
+*(Note du traducteur: Le gestionnaire pip crée des liens logiques vers les versions de Python. En effet, si votre ordinateur inclut initialement la version 2.x de Python, la commande `pip` pointera alors sur cette version de Python. En revanche, si vous décidez d'installer Python en version 3.x en plus de Python 2.x, il vous faudra très certainnement utiliser la  commande `pip3` dans votre terminal et non plus `pip` pour installer des bibliothèques pour Python 3.x. Afin de connaitre quelle version de Python vous utilisez avec la commande `pip`, lancer dans le terminal la commande `pip -V` ou `pip --version`*
```
Comment (mariechristineb) [inline fr/lecons/installation-modules-python-pip.md]: ici, j'écrirais plutôt "lancez dans le terminal" (ou "il suffit de lancer dans le terminal")

Comment (mariechristineb) [reply fr/lecons/installation-modules-python-pip.md]: Et il semble manquer un point à la fin du paragraphe, juste après "`pip --version`*"

**Review by mariechristineb — DISMISSED** [review]

**Comment by spapastamkou** [comment]
Thanks @programminghistorian/technical-team, it is fixed. 
When @svmelton / @programminghistorian/english-team can review the changes in https://github.com/programminghistorian/jekyll/pull/1973/commits/755ca098e317dc81e1e9c8b76e5b055ea76ee2e3
and @rivaquiroga / @programminghistorian/spanish-team the changes in https://github.com/programminghistorian/jekyll/pull/1973/commits/e3419f0d8627c0a06a8d54c5f80b524f0326bb92 this can be  merged.

**Review by rivaquiroga — APPROVED** [review]

**Review by svmelton — APPROVED** [review]

**Pull request merged**